css/properties/hyphens.json

```diff
@@ -794,10 +794,10 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": null
+                "version_added": true
```

Comment:
The PR description made it seem like the orthography features would be `false`. Did I misunderstand something here?

Reply:
You didn't -- I just typed `true` on accident...  Durr.  -_-